safe_core/src/client/mock/vault.rs

```diff
@@ -173,7 +173,7 @@ impl Vault {
             Some(balance) => balance,
             None => {
                 debug!("Account not found for {:?}", coin_balance_name);
```

Comment:
Should be
```suggestion
                 debug!("Coin balance not found for {:?}", coin_balance_name);
```
for consistency?